### PR TITLE
Completed Typescript and JSON Exercise-Allyson Greven

### PR DESCRIPTION
**Pull request by allygreven**
Screenshot 2024-11-08 at 4 16 25 PM (2)

### CODE REVIEW

File: typescript-and-json/main.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const array = {
+  isbn: '9798352831533',
+  title: 'Never Lie',
+  author: 'Frieda McFadden',
+};
```
Comment (shawnkost) [inline typescript-and-json/main.ts:5]: Is this an array or an object? This should be an array of objects

File: typescript-and-json/main.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+console.log('value of isbn:', typeof '9798352831533');
+console.log('value of title:', typeof 'Never Lie');
+console.log('value of author:', typeof 'Frieda McFadden');
```
Comment (shawnkost) [inline typescript-and-json/main.ts:9]: These should just be logging your array and the `typeof array`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Ally!

**Comment by allygreven** [comment]
Screenshot 2024-11-19 at 1 57 39 PM (2)

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**